Mage/src/main/java/mage/abilities/hint/ValueConditionHint.java

```diff
@@ -0,0 +1,42 @@
+package mage.abilities.hint;
+
+import mage.abilities.Ability;
+import mage.abilities.condition.Condition;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.game.Game;
+
+/**
+ * @author notgreat
+ */
+public class ValueConditionHint implements Hint {
```

Comment:
Good idea :)

Reply:
Must extends ConditionHint. See example with EvidenceHint in e7b8eecd64c5aabd5659ac5dff65fdf34ca68675